bedrock/mozorg/redirects.py

```diff
@@ -572,6 +572,9 @@ def decider(request, **kwargs):
     # bug 1003737
     redirect(r'^impressum/?$', '/about/legal/impressum/'),
 
+    # bug 1236910
+    redirect(r'^support(/.*)?$', 'https://support.mozilla.org/'),
```

Comment:
This should catch it with or without a locale prefix. I thought that would be best.